modules/massrole/commands/massrole.js

```diff
@@ -0,0 +1,202 @@
+const {localize} = require('../../../src/functions/localize');
+let target;
+
+module.exports.subcommands = {
+    'add': async function (interaction) {
+        await checkTarget(interaction);
```

Comment:
```suggestion
        checkTarget(interaction);
```
This function is not asyncron and does not need await